core/sector_storage/impl/local_worker.cpp

```diff
@@ -272,13 +273,12 @@ namespace fc::sector_storage {
       gsl::span<const UnpaddedPieceSize> piece_sizes,
       const UnpaddedPieceSize &new_piece_size,
       proofs::PieceData piece_data) {
-    std::function<outcome::result<PieceInfo>()> work =
-        [self{shared_from_this()},
-         sector,
-         piece_sizes,
-         new_piece_size,
-         piece_data{std::make_shared<PieceData>(
-             std::move(piece_data))}]() -> outcome::result<PieceInfo> {
+    return asyncCall(
+        sector,
+        return_->ReturnAddPiece,
+        [=, piece_data{std::make_shared<PieceData>(std::move(piece_data))}](
+            Self self) -> outcome::result<PieceInfo> {
+          // clang-format off
```

Comment:
```suggestion
```
these `// clang-format` comments were added to simplify diff.
should be removed and formatted before merging to master.